uint32 plan_id
uint32 sync_id
WaypointItem[] items
